client/my-sites/checkout/composite-checkout/record-analytics.js

```diff
@@ -312,6 +313,22 @@ export default function createAnalyticsEventHandler( reduxDispatch ) {
 					// TODO: Decide what to do here
 					return;
 				}
+				case 'WPCOM_EMAIL_ALREADY_EXISTS': {
```

Comment:
While this is fine to do this here (and makes sense given the central location), if these events are only called in one place each, it might be just as easy and more clear to record them directly where they occur. The redirection of this handler was mainly to allow recording events triggered by non-calypso packages, and I've actually been trying to remove as much of it as possible ([one attempt at this which had to be scrapped](https://github.com/Automattic/wp-calypso/pull/48282)).

Reply:
Ahh I see, yes makes sense. I agree they will be more clear to record them directly where they occur. I'll move them out of the handler. 👍